src/routes/GroupExplorer/GroupSearch/GroupSearch.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react'
+import cx from 'classnames'
+import { useSelector } from 'react-redux'
+import Dropdown from 'components/Dropdown'
+import Icon from 'components/Icon'
+import Loading from 'components/Loading'
+import NoPosts from 'components/NoPosts'
+import ScrollListener from 'components/ScrollListener'
+import GroupCard from 'components/GroupCard'
+import { CENTER_COLUMN_ID } from 'util/scrolling'
+import './GroupSearch.scss'
+import { SORT_NAME, SORT_NEAREST, SORT_SIZE } from 'store/constants'
+import useRouter from 'hooks/useRouter'
+import useDebounce from 'hooks/useDebounce'
+import useEnsureSearchedGroups from 'hooks/useEnsureSearchedGroups'
```

Comment:
this is the order i try to use for imports:

first all third party modules, alphabetized
Then our files from other directories, also alphabetized (so components, then hooks, then store, then util...)
then files from own directory, with css file coming last.

Reply:
We have somehow converged on this pattern overtime ^^. It's hard to encapsulate in a brief description and you've done an admirable job of it @tibetsprague :) Will save this somewhere for our contribution guide. Def. a "guideline" not a rule in my book (e.g. not something particuarly lint;'able), but I really like it when things are ordered according to this particular formula. Helps my eyes and brain.